alf/algorithms/one_step_loss.py

```diff
@@ -41,3 +44,139 @@ def __init__(self,
             debug_summaries=debug_summaries,
             td_lambda=0.0,
             name=name)
+
+
+@alf.configurable
+class OneStepTDQRLoss(OneStepTDLoss):
+    """One step temporal difference quantile regression loss. """
+
+    def __init__(self,
+                 num_quantiles=50,
```

Comment:
The `num_quantiles` can be inferred from input, seems like we do not need to pre-specify it in the initialization.

Reply:
I feel that the `num_quantiles` is fixed throughout our calls of the `OneStepTDQRLoss`, so I want to make the `cdf_midpoints` fixed at initialization, rather than recreating it at every calls.

Reply:
That make senses. We can also do the
`self._cdf_midpoints = self._cdf_midpoints.view(1, 1, 1, -1)`
in the initialization.

And add an assertion to make sure the number of quantiles of the input is the same as the pre-defined `num_quantiles`, i.e., `assert value.shape[-1] == self._num_quantiles`

Reply:
I added the assertion and left the unsqueezing of cdf_midpoint here to easily incorporate the multi-dim reward case.